pkg/telemetry/manager.go

```diff
@@ -37,6 +39,9 @@ const (
 )
 
 type manager struct {
+	// TODO
```

Comment:
What are we doing about this TODO? What is it here for?

Reply:
Leftovers from marking it for myself, removed.